spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/convert/TestDatastoreItem2.java

```diff
@@ -0,0 +1,44 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.data.datastore.core.convert;
+
+import java.math.BigInteger;
+
+/**
+ * @author Dmitry Solomakha
+ */
+class TestDatastoreItem2 {
```

Comment:
This one seems like a class that is supposed to cause problems in the tests , right ? I think it should be named "TestDatastoreItemUnsupportedFields" or something. I think that would make more sense  when BigInteger and other such fields will be supported (because then there would look like there are 2 working test classes like this)